kiva/fonttools/_util.py

```diff
@@ -44,10 +44,80 @@
     5: "version",
     6: "postscript_name",
 }
+# OpenType 'OS/2' `ulCodePageRange` bit meanings
+_ot_code_page_masks = {
+    "Latin": 0x93,
+    "Cyrillic": 0x4,
+    "Greek": 0x8,
+    "Hebrew": 0x20,
+    "Arabic": 0x40,
+    "Vietnamese": 0x100,
+    "Thai": 0x10000,
+    "Japanese": 0x20000,
+    "Simplified Chinese": 0x40000,
+    "Traditional Chinese": 0x100000,
+    "Korean": 0x280000,
+    "Symbol": 0x80000000,
+}
+# OpenType 'OS/2' table `ulUnicodeRange` bit meanings
```

Comment:
and another reference here, like - https://docs.microsoft.com/en-us/typography/opentype/spec/os2#ur